<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="2">

  <testsuite name="lvlOne"
    errors="1" failures="0" hostname="" id="" package="lvlOne" skipped="1"
    tests="2" time="0.005000" timestamp="Fri, 20 Jul 2018 20:34:32 GMT">
  
    <testcase name="login" classname="lvlOne" time="0.005000" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Lvl One / login]:

  ReferenceError: loginData is not defined
    at Object.login (C:\Users\Maximus Prime\Desktop\QA-Mountain\Cards\test\lvlOne.js:19:38)
    at Module.call (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:62:34)
    at C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:70:29
    at _fulfilled (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\Maximus Prime\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at process._tickCallback (internal/process/next_tick.js:150:11)
    </system-err>
  

  
    
    <testcase
      name="Decks, Subscription, Contact Us, Settings, Logout" classname="lvlOne">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
